BunqSdk/Json/BunqContractResolver.cs

```diff
@@ -28,6 +31,20 @@ public BunqContractResolver()
             RegisterConverter(typeof(double?), new NonIntegerNumberConverter());
             RegisterConverter(typeof(float?), new NonIntegerNumberConverter());
             RegisterConverter(typeof(Pagination), new PaginationConverter());
+            RegisterConverter(typeof(IAnchorObjectInterface), new AnchorObjectConverter());
+
+            if (typesToExclude == null)
+            {
+                return;                
+            }
+            
+            foreach (var type in typesToExclude)
+            {
+                if (converterRegistry.ContainsKey(type))
```

Comment:
No need to check if the key is present: `Dictionary.Remove()`. If the key is not found, it returns `false` but no exception is thrown